README.md

```diff
@@ -11,64 +11,47 @@ pip install grpcio-opentracing
 
 ## Getting started
 
-See the below code for basic usage or [examples/trivial](examples/trivial) for a
-complete example.
+using official grpc interceptor
```

Comment:
I think we could leave the original line ("See the below code...") and also mention this new note at the end of the paragraph ;)

Reply:
Agreed.